main.py

```diff
@@ -1,10 +1,58 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+
+    for player_name, player_data in players.items():
+        race = player_data["race"]
+        guild = player_data["guild"]
+        skills = race["skills"]
+
+        race_name = race["name"]
+        if not Race.objects.filter(name=race_name).exists():
+            race_description = race["description"] \
+                if race["description"] else ""
+            Race.objects.create(
+                name=race_name,
+                description=race_description,
+            )
+        race_obj = Race.objects.get(name=race_name)
```

Comment:
You can use get_or_create, it will be more optimized
Fix also in other places
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create